core/chains/evm/client/errors.go

```diff
@@ -469,7 +472,9 @@ func ClassifySendError(err error, lggr logger.Logger, tx *types.Transaction, fro
 		return commonclient.InsufficientFunds, err
 	}
 	if sendError.IsTimeout() {
-		return commonclient.Retryable, errors.Wrapf(sendError, "timeout while sending transaction %s", tx.Hash().Hex())
+		errorMsg := fmt.Sprintf("timeout while sending transaction %s", tx.Hash().Hex())
+		lggr.Errorw(errorMsg, "err", sendError)
+		return commonclient.Retryable, fmt.Errorf("%s: %w", errorMsg, sendError)
```

Comment:
Why do we wrap `sendError` here? There is only one other case like this. All the others just returns `err`. If we could use pure `err` in all cases, then we could simplify this whole func by dropping it from the return args.
```suggestion
		return commonclient.Retryable
```

Reply:
Looking at all of the places this is called, I don't think there's any use to wrap `sendError` here. It's the same as the passed in error with an additional fatal flag that's only used internally anyways. I've updated the PR to clean this up